.circleci/config.yml

```diff
@@ -188,6 +188,20 @@ jobs:
             make plugin-checker
             make vet BUILD_NUMBER='${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM}' MM_NO_ENTERPRISE_LINT=true MM_VET_OPENSPEC_PATH='${PWD}/../mattermost-api-reference/v4/html/static/mattermost-openapi-v4.yaml'
 
+  # Dedicate job for sentry it does not need anything only the server code for that
+  # and to make more clear when the job fails
+  sentry:
+    docker:
+      - image: getsentry/sentry-cli:1.64.1
+    steps:
+      - checkout
```

Comment:
Reading from the workspace should still be fine and might be a bit quicker than a full checkout, but I don't have anything against this either

Reply:
sometimes take more time to get the workspace back